addons/project_timesheet_holidays/models/project_task.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+class Task(models.Model):
+    _inherit = 'project.task'
+
+    timeoff_type_ids = fields.One2many('hr.leave.type', 'timesheet_task_id', string="Task Time off Types")
+    is_timeoff_task = fields.Boolean(string="Is Time off Task", compute="_compute_is_timeoff_task",
+        search="_search_is_timeoff_task")
```

Comment:
I think it could be written in one line. 🙂 
```suggestion
    is_timeoff_task = fields.Boolean("Is Time off Task", compute="_compute_is_timeoff_task", search="_search_is_timeoff_task")
```